src/Core/Domain/Order/Payment/Command/AddPaymentCommand.php

```diff
@@ -89,10 +88,9 @@ public function __construct(
         $paymentAmount,
         $paymentCurrencyId,
         $orderInvoiceId = null,
-        $transactionId = ''
+        $transactionId = null
```

Comment:
Should be an int like $orderInvoice :thinking: 
```suggestion
        ?int $transactionId = null
```

Reply:
This transaction id is not related to PS object model, it can be any transation id from bank (e.g. UUID).

Reply:
Could you improve the comment in the php doc block please? :)